bin/porta

```diff
@@ -375,27 +375,11 @@ module Porta
     protected
 
     def docker_internal_host
-      options[:docker_internal_host] ||= container_local_host_ip
+      "127.0.0.1"
     end
 
-    def container_local_host_ip(image: nil)
-      gw_ip_cmd = %{printf "%d.%d.%d.%d" $(awk '$2 == 00000000 && $7 == 00000000 { for (i = 8; i >= 2; i=i-2) { print "0x" substr($3, i-1, 2) } }' /proc/net/route)}
-      `docker run --rm --entrypoint /bin/bash #{image || options[:porxy_image]} -c #{gw_ip_cmd.shellescape}`
-    end
-
-    def container_opts_local_hosts(*hosts)
-      unless macos?
-        hosts << "host.docker.internal"
-        if `docker --version`.include?("podman")
-          gw_ip = docker_internal_host
-          host_access = "--net slirp4netns:allow_host_loopback=true,enable_ipv6=false"
-          if File.read("/etc/resolv.conf") =~ /nameserver\s+127\.0\.0\.1$/
-            host_access += " --dns=#{gw_ip}"
-          end
-        end
-      end
-      gw_ip ||= 'host-gateway'
-      hosts.map { |h| "--add-host=#{h}:#{gw_ip} " }.join + "#{host_access}"
+    def container_opts_local_hosts
+      "--network=host"
```

Comment:
Does host network work with rootless containers?  I'm not sure this is the right solution. But good to know my procrastination to upgrade to 40 is not unfounded...

Reply:
> Does host network work with rootless containers?

It does.

> I'm not sure this is the right solution

It doesn't intend to. I'm just sharing my workaround, in case somebody updates to Fedora 40 they don't have to spend time investigating this like I did.